client/components/parentComponents/Transactions.jsx

```diff
@@ -0,0 +1,63 @@
+import React, { useState, useEffect } from 'react';
+import { connect } from 'react-redux';
+
+import axios from 'axios';
+
+const Transactions = (props) => {
+  const [transactions, setTransactions] = useState([]);
+  const days = {
+    0: 'Sun',
+    1: 'Mon',
+    2: 'Tues',
+    3: 'Wed',
+    4: 'Thurs',
+    5: 'Fri',
+    6: 'Sat',
+  };
+
+  let card = 'ic_1IzufNGMLeOpoTZxPd1bYRNy';
```

Comment:
Does this mean the card is hard coded into the program?

Reply:
It's just for testing purposes. Once the parent landing page is complete, we will check which child card we are currently on and get the virtual card id from there.